\documentclass[12pt]{article}

\usepackage{setspace}
\doublespacing

\usepackage{natbib}
\bibliographystyle{apalike}

\usepackage[margin=1.0in]{geometry}

\usepackage{amsmath}

\usepackage{lineno}

\usepackage{graphicx}
\linenumbers

%opening
\title{Predicting detection probabilities for rare and undersampled North American landbirds}
\author{
	Edwards, Brandon P.M.\\
	\and
	Solymos, Peter\\
	\and
	Stralberg, Diana\\
	\and
	Michel, Nicole\\
	\and
	Robinson, Barry\\
	\and
	Gahbauer, Marcel\\
	\and
	Grinde, Alexis\\
	\and
	Hope, David\\
	\and
	Bennett, Joseph R.\\
	\and
	Smith, Adam C.\\
}

\begin{document}

\maketitle

\begin{abstract}

\end{abstract}

\section{Introduction}

\par Accurate and precise estimates of detectability are important both for making efficient conservation decisions, and estimating accurate population sizes.
In conservation management situations, detectability can be used as a metric of sensitivity to evaluate when to monitor a system for a species, or when to act on current information \citep{canessa_when_2015, bennett_when_2018}. 
When estimating population sizes of an organism, detectability often comes into play as a way to convert observed counts into densities or estimates of true abundance \citep{solymos_calibrating_2013, johnson_defense_2008}.
However, despite its importance, detectability often goes unaccounted for in studies, or detectability data just does not exist \citep{bennett_how_2023}.
The wealth of openly available bird data in North America means that estimates of detectability for birds are relatively easy to come by, in comparison to most other taxonomic groups \citep{bennett_how_2023}. 
The NA-POPS project is an effort to estimate detectability functions that take into account variability in detection through environmental and temporal covariates, for as many of North America’s landbird species as possible \citep{edwards_point_2023}.
To date, the project has estimated availability (the probability a bird sings or gives a cue) and perceptibility (the probability an observer sees or hears a cueing bird) for 319 species of birds, with an additional 19 species having estimates of availability or perceptibility (but not necessarily both). This project built off previous efforts by the Boreal Avian Modelling project \citep{cumming_toward_2010} to systematically estimate detectability in landbirds \citep{solymos_calibrating_2013, solymos_evaluating_2018}.
Both the NA-POPS project and previous estimates by Boreal Avian Modelling project use the QPAD methodology \citep{solymos_calibrating_2013}, which is a flexible method to estimate availability and perceptibility given a harmonized dataset \citep{barker_ecological_2015}.

\par Despite the efforts of the NA-POPS project, approximately 25\% of the species in the Partners in Flight’s 2016 Landbird Conservation Plan \citep{rosenberg_partners_2016} have insufficient data to directly estimate detectability \citep{edwards_point_2023}.
This is because of sample size limitations that are recommended for removal and distance modelling, in that a modeller should have at least 75 data points for reasonable estimates \citep{buckland_introduction_2001, solymos_calibrating_2013}. 
Many of the species that fell short of that threshold in the NA-POPS database are rare or undersampled species, that are either have low population sizes, or occur in rare or inaccessible habitats.
Some examples of species not modelled include Kirtland’s Warbler (\textit{Setophaga kirtlandii}) and Bicknell’s Thrush (\textit{Catharus bicknelli}), both of which have low population sizes, and LeConte’s Thrasher (\textit{Toxostoma lecontei}) and Harris’s Sparrow (\textit{Zonotrichia querula}), both of which have rare habitat associations \citep{will_handbook_2020}. 
Such species are often conservation concerns, and thus detectability information for them would be particularly important.


\par Detectability in birds has been shown to be at least partly driven by variation in traits.
\cite{johnston_species_2014} showed in a selection of European landbirds that body mass and habitat association had an effect on detection distance, which influences perceptibility
In this case, a larger body size tended to be associated with larger detection distances, which makes sense considering larger body sizes are associated with louder and lower-pitched songs in birds \citep{bowman_adaptive_1979, fletcher_acoustics_1999, ryan_role_1985}.
Habitat associations tend to influence the attenuation of sound, whereby sound diminishes quicker in a closed-forested habitat than more open habitat \citep{waide_tropical_1988, yip_sound_2017}.
A study by \cite{solymos_phylogeny_2018} expanded on this effort by also considering the effects of species traits and phylogeny on both detection distance and cue rate, the latter of which influences the availability of birds.
They found that cue rate is most related to phylogeny, and, to a lesser extent, whether a bird was a long-distance migrant or a resident species.
They also found detection distance was related to species’ traits such as body mass, song pitch, habitat association, and migratory status and that after accounting for these traits, phylogeny explained very little.

\par One recommendation for future research that came out of \cite{solymos_phylogeny_2018} was to use these phylogenetic and trait relationships with detectability to predict components of detectability (i.e., availability through cue rate, and perceptibility through detection distance) in other birds.
Indeed, by making use of shrinkage factors and partial pooling, as can be done in Bayesian hierarchical models \citep{gelman_what_2021}, and by accounting for phylogenetic relationships and the relative "relatedness" of different species, as can be done with Gaussian processes \citep{bernardo_regression_1998, mcelreath_continous_2020}, we can leverage the information provided by species with many data to inform estimates for species with much fewer (included potentially zero) data points.

\par The goal of this paper is to create a multi-species model of detectability that incorporates species traits and phylogeny, to both allow for the relaxation of the sample size requirements for estimating detectability, and to derive estimates of detectability for bird species without any data. 
To do so, we developed a Bayesian hierarchical version of the QPAD Removal model initially derived in \cite{solymos_calibrating_2013}, that allows for information exchange between species based on phylogenetic distance and binned migratory status.
Phylogenetic distance was coded as a correlation structure and used as a Gaussian random field in this model, and migratory status was coded similar to \cite{solymos_phylogeny_2018} where a species was considered either "migrant" or "resident".
We also developed a Bayesian hierarchical version of the QPAD Distance model initially derived in \cite{solymos_calibrating_2013}, that allows for information exchange between species based only on the traits that came out as being significant in  \cite{solymos_phylogeny_2018}.
These traits include migratory status (coded the same as removal models), habitat association (coded as either "open" or "closed"), song pitch, and body mass. 
We also developed a Bayesian single species version of the QPAD Removal and Distance models (i.e., as opposed to the maximum likelihood models available in the \texttt{detect} R package; \cite{solymos_detect_2020}) to allow for direct comparison through cross validation.
Finally, we ran these models with the full set of data points used in \cite{edwards_point_2023}, plus additional data for seven species that did not meet the initial sample size thresholds for NA-POPS: LeConte's Thrasher, Bicknell's Thrush, Lesser Prairie-chicken (\textit{Tympanuchus pallidicinctus}), Harris's Sparrow, Kirtland's Warbler, Tricoloured Blackbird (\textit{Agelaius tricolor}), and Spotted Owl (\textit{Strix occidentalis}). 
All seven of these bird species are listed on Partners in Flight's watchlist for various threats \citep{will_handbook_2020}. 

\par We predicted that for both the Bayesian hierarchical QPAD removal model and Bayesian hierarchical QPAD distance model, estimates of cue rate and EDR, respectively, will change very little from those estimated from the original NA-POPS database \cite{edwards_point_2023}. 
This is because the species that were modelled in \citet{edwards_point_2023} had a sufficient number of data points to stand on their own, and so adding in species-level effects should have little effect on estimates. 
Furthermore, we predicted that improvement in precision from a multi-species model will be related to sample size, such that precision of estimates will improve the most for species with smaller sample size, while precision of estimates will improve the least for species with larger sample sizes.
Finally, we predicted that a multi-species model will have greater predictive capacity than a single-species model, particularly for species which are data-sparse.
With the predictions above, we then expect to be able to generate estimates of cue rate and EDR for species with very little data, thus relaxing previous sample size requirements; additionally, we expect to be able to predict cue rate and EDR for species with no data by making use of the shrinkage effect offered by a multi-species model.

\section{Methods}

\par The following subsections outline the modelling techniques used in this study as well as the data that were collected. 
We start by introducing the Bayesian hierarchical QPAD removal and distance models, and then describe the input.
Finally, we describe the experiments conducted to assess model performance and model comparison, which then allow us to make predictions for the seven undersampled species of interest.

\subsection{Bayesian Hierarchical Models}

\par For any point count that employs removal sampling \citep{alldredge_time--detection_2007, farnsworth_removal_2002} and/or distance sampling \citep{buckland_introduction_2001, buckland_distance_2015}, let $Y_{sijk}$ be the observed count of species $s$ during sampling event $i$, occurring in time band $j \in [1,J]$ and/or distance band $k \in [1,K]$.
That is, if we have a point count where removal sampling or distance sampling (but not both) is not used, then either $J = 1$ or $K = 1$, respectively; whereas, if we have a point count where both removal sampling and distance sampling is used, then both $J,K \geq 2$.
The case where $J = K = 1$ is where neither removal sampling nor distance sampling is used, and so these data points are not appropriate for the modelling techniques described here.

\subsubsection{Bayesian Hierarchical QPAD Removal Model}

\par For removal modelling, we are only interested in the time bin in which the bird was recorded, and not the distance, and so we will sum counts from each sampling event $i$ over all distance bands. 
Thus, we will be considering counts $Y_{sij.} = \sum_{k=1}^{K}$. 
From \citet{solymos_calibrating_2013}, we model $Y_{sij.}$ as
$$Y_{sij.} \sim multinomial\left(Y_{si..}, \mathbf{\Pi}_{si.}\right).$$

\par For sample $i$, $\mathbf{\Pi}_{si.}$ is the corresponding probability vector that determines how the total number of species $s$ observed at sample $i$ (i.e., $Y_{si..}$) are to be distributed across the $J$ time bands; in essence, it is the mixing parameter.
Let $t_{ij}$ be the maximum time for time band $j$ during sampling event $i$, and let $\phi_s$ be the unknown cue rate for species $s$.
We then have the following calculation of each component $\pi_{sij}$ of the mixing parameter $\mathbf{\Pi}_{si.}$:

\begin{equation*}
	\pi_{sij} = 
	\begin{cases}
		\dfrac{\exp\left\{ -t_{i,j-1}\phi_{s} \right\} - \exp\left\{ -t_{ij}\phi_{s} \right\}}{1 - \exp\left\{ -t_{iJ}\phi_{s} \right\}} & \text{for } j > 1 \\
		1 - \sum_{n = 2}^{J} \pi_{sin} & \text{for } j = 1
	\end{cases}
\end{equation*}

\par We are most interested in estimating $\phi_s$, the cue rate for species $s$. 
To allow for the partial pooling of estimates for each species to be based on a phylogenic relationship, we can use Gaussian processes, which effectively allow for "continuous" factors \citep{bernardo_regression_1998, mcelreath_continous_2020}. 
Let $\mathbf{\Phi}$ be the vector of cue rates (of length $s$). 
We then have:

$$\log \mathbf{\Phi} \sim MVN\left( \mathbf{\mu}_{[s]}, \mathbf{\Sigma}_{[s \times s]} \right).$$

\par The covariance function $\mathbf{\Sigma}$ (denoted with subscript $[s \times s]$ to show dimensionality) determines the strength of the partial pooling, and can be any function that contains a measure of distance. 
In this case, we will create a function based on phylogenetic distance $\mathbf{D}_{[s \times s]}$, i.e.,

$$\mathbf{\Sigma}_{[s \times s]} = \mathbf{D}_{[s \times s]}\lambda\sigma.$$

\par Here, $\lambda$ is Pagel's Lambda \citep{pagel_inferring_1999}, which determines the strength of the phylogenetic relationship. 
We set $\lambda = 0.76$, consistent with previous results \cite{solymos_phylogeny_2018}. 
We also estimated the extra variance term and used an exponential prior $\sigma \sim exp(5)$.

\par Finally, the vector of mean cue rate $\mathbf{\mu}_{[s]}$ is a function of the species migratory strategy $\alpha_M$, where the migratory strategy $M$ could either be Resident or Migrant. 
We set priors on both migratory strategies to be $\alpha_M \sim N(-1, 0.01)$, because on the log scale we would expect cue rate to be negative. 

\subsubsection{Bayesian Hierarchical QPAD Distance Model}

\par For distance modelling, we are only interested in the distance to the cueing bird, and not the time bin that the bird was recorded in, and so we will sum counts from each sampling event $i$ over all time bands. 
Thus, we will be considering counts $Y_{si.k} = \sum_{j=1}^{J}$. 
From \citet{solymos_calibrating_2013}, we model $Y_{si.k}$ as
$$Y_{si.k} \sim multinomial\left(Y_{si..}, \mathbf{\Pi}_{si.}\right).$$

\par For sample $i$, $\mathbf{\Pi}_{si.}$ is the corresponding probability vector that determines how the total number of species $s$ observed at sample $i$ (i.e., $Y_{si..}$) are to be distributed across the $K$ distance bins; in essence, it is the mixing parameter. 
Let $r_{ik}$ be the maximum distance for distance bin $k$ during sampling event $i$, and let $\tau_s$ be the unknown effective detection radius for species $s$. 
We then have the following calculation of each component $\pi_{sik}$ of the mixing parameter $\mathbf{\Pi}_{si.}$:

\begin{equation*}
	\pi_{sik} = 
	\begin{cases}
		\dfrac{f(r_{i,k}, \tau_s) - f(r_{i,k-1}, \tau_s)}{f(r_{i,K}, \tau_s)} & \text{for } k > 1 \\
		1 - \sum_{n = 2}^{K} \pi_{sin} & \text{for } k = 1
	\end{cases}
\end{equation*}

where 
$$f(r,\tau) =  1 - \exp\left\{ -\dfrac{r^2}{\exp\left\{\tau^2\right\}} \right\} .$$

\par We are most interested in estimating $\tau_s$, the effective detection radius for species $s$. 
To allow for the partial pooling of estimates for each species to be based on traits, we set up a mixed effects model as follows:
$$\log \tau_s \sim N(\mu_s, \sigma).$$

\par In this model, mean effective detection radius for species $s$ (i.e., $\mu_s$) is a function of the following traits as described in \cite{solymos_phylogeny_2018}: an overall intercept term $\alpha_0$; migratory strategy $\alpha_M$, where $M$ can be Migratory or Resident; habitat preference $\alpha_H$, where $H$ can be Open or Closed habitat; log body size $\beta_b$; and log song pitch $\beta_p$. 
That is, we have
$$ \mu_s = \alpha_0 + \alpha_{M_s} + \alpha_{H_s} + \beta_b \times \log BodySize_s + \beta_p \times \log SongPitch_s$$
$$\alpha_0 \sim N(0.05, 0.1)$$
$$ \alpha_M, \alpha_H \sim N(0, 0.05)$$
$$ \beta_b \sim N(0.01, 0.005)$$
$$ \beta_p \sim N(-0.01, 0.005) $$
$$\sigma \sim exp(5)$$

\subsection{Single Species Models}

\par To directly compare the multi-species models described above to its single-species equivalent in a cross-validation (see below), we built two Bayesian single-species models in Stan.
The single species models used the same likelihood statements as above for the count data, except rather than cue rate and EDR being estimated with shared information, they are estimated independently from each other.
That is, $\log \phi_s \sim N(0,1) \forall s$, and $\log \tau_s \sim N(0,1) \forall s$, thus making it equivalent to the Null model in \citet{edwards_point_2023}, just estimated in a Bayesian framework rather than maximum likelihood. 

\subsection{Data Collection}
\subsubsection{Point Count Data}
\par We used the same standardized database of point count data that was assembled through the NA-POPS project \cite{edwards_point_2023}. 
This consists of $712138$ point counts conducted across $292$ projects in Canada and the United States, $422514$ of which used removal sampling and $522820$ of which used distance sampling.
All of these point counts were recorded in the database in a standard way, using practices developed by the Boreal Avian Modelling project \citep{barker_ecological_2015}.

\subsubsection{Phylogenetic Trees}
\par We accessed 2000 pseudo-posterior trees from \citet{jetz_global_2012} for all species with sufficient data for removal modelling ($>75$ observations as per \cite{edwards_point_2023,solymos_evaluating_2018,buckland_introduction_2001}). 
We also requested that the phylogenies include the following bird species with insufficient data from NA-POPS: LeConte's Thrasher, Bicknell's Thrush, Lesser Prairie-chicken, Harris's Sparrow, Kirtland's Warbler, Tricoloured Blackbird, and Spotted Owl. 
All seven of these bird species are listed on Partner's in Flight's watchlist for various threats \citep{will_handbook_2020}. 
This resulted in 2000 pseudo-posterior trees that contained 323 species.

\par We generated a phylogenetic correlation matrix $\mathbf{D}$ using the same methods outlined in the Appendix of \citet{solymos_phylogeny_2018} that generate distances between species, and further processed using the R package "ape" \citet{paradis_ape_2019}.

\subsubsection{Species Traits}
\par Species traits (song pitch, migratory vs resident, open vs. closed habitat) were obtained from Birds of the World accounts of all the birds to model in this study \citep{billerman_birds_2022}. 
Body mass for each bird species was obtained from the Elton traits dataset \citep{wilman_eltontraits_2014}. 
Where body mass was not available through the Elton traits dataset, the Birds of the World account \citep{billerman_birds_2022} was used instead.

\subsection{Model Assessment}

\par To test prediction \ref{pred1}, we ran the Bayesian hierarchical QPAD Removal and Distance models on the set of species for which there was sufficient data in the original NA-POPS analysis.
In this case, we ran both models for 316 species of birds.
Rufous Hummingbird (\textit{Selasphorus rufus}) was dropped from the distance modelling due to issues with divergent transitions \citep{betancourt_diagnosing_2016, leimkuhler_simulating_2005}.
Tricoloured Blackbird had sufficient distance data, and so was included in the distance modelling, but not the removal modelling.
We accessed the single-species estimates of cue rate and EDR from the NA-POPS Null models using the \texttt{napops} R package \citep{edwards_napops_2023, edwards_point_2023}.
We then modelled the difference in estimated cue rates and EDRs as
$$\Delta X \sim N(0,\sigma)$$
$$\sigma \sim exp(1)$$

where $\Delta X$ is calculated as the mean multi-species estimate of cue rate or EDR minus the mean single-species estimate of cue rate of EDR.
If prediction \ref{pred1} is true, we would expect to see a distribution of modelled differences centred about 0, indicating no real difference in estimates of cue rate or EDR between single-species and multi-species models, for species for which we have sufficient data.

\par To test prediction \ref{pred2}, we modelled the standard deviations about each estimate of cue rate or EDR from both the multi-species and single-species models, and regressed them against the log sample size per species.
That is
$$SD \sim N(\alpha_M + \beta_M\times \log(N), \sigma)$$
$$\alpha, \beta \sim N(0,1)$$
$$\sigma \sim exp(1)$$

where $\alpha$ is the model-specific intercept and $\beta$ is the model-specific slope.
In both cases, $M=1$ corresponds to the single species model and $M=2$ corresponds to the multi-species model.

\par If prediction \ref{pred2} is true, we would expect to see a higher value of $\alpha_1$ than for $\alpha_2$, corresponding to a higher baseline standard deviation in the single species models.
We would also likely see a slightly smaller (i.e., more negative) value of $\beta_1$ than $\beta_2$ (which should also be negative), indicating that the standard deviation decreases with log sample size for both models, but should decrease at a slower rate with the multi-species model (which has a lower baseline standard deviation to begin with) than the single-species model.

\par To test prediction \ref{pred3}, we ran a k-fold cross-validation with 5 folds, stratified by species. That is, for each cross-validation run, we held out 20\% of the data for each species for both the multi-species and Bayesian single-species models. 
We then derived multi-species estimates of cue rate and EDR and single-species estimates of cue rate and EDR for each cross-validation run, and used these parameter estimates to calculate the posterior predictive density of each of the held out data points. 
We then calculated the difference in posterior predictive density for removal and distance models, by subtracting the pointwise posterior predictive density of the single-species model from the pointwise posterior predictive density of the multi-species model (i.e., multi-species minus single-species).
Finally, we took the mean of the difference in pointwise posterior predictive density for each species to create a species-specific mean difference in pointwise predictive density.
With this, positive values would indicate that the multi-species model was preferred, and negative values would indicate that the single-species model was preferred.
If prediction \ref{pred3} is true, then we would expect to see majority positive values of the species-specific mean pointwise predictive density.


\subsection{Predicting Detection Probabilities}
\par To predict estimates of cue rate or EDR from the multi-species models, we adopted two possible approaches.
If the NA-POPS database contained some point-count data for the species of interest, we simply added these data into the model.
This is different from \citet{edwards_point_2023, solymos_calibrating_2013, solymos_evaluating_2018}, where any species with less than 75 data points were removed from the analysis.
For species that did not have any data in the NA-POPS database, we generated a row of traits data for the species and estimated its cue rate and EDR as parameters in the model.
We then ran the multi-species models described above using these additional data points (and hence an updated phylogenetic correlation structure for the removal model) to generate estimates of cue rate and EDR for these new species.

\subsection{Analysis}
\par All analyses were performed in R version 4.2.2 \citep{r_core_team_r_2022}.
Manipulation of phylogenetic trees, including finding a consensus tree for visualization and generating variance-covariance matrices based on the 2000 bootstrapped trees, was done using the \texttt{ape} R package \citep{paradis_ape_2019}.
All Bayesian models were coded in the Stan Probabilistic Language \cite{stan_development_team_stan_2019}, and models were run using the \texttt{cmdstanr} R package \citep{gabry_cmdstanr_2022} and visualized using the \texttt{bayesplot} R package \citep{gabry_visualization_2019}.
Each of the single-species models and multi-species models running both the initial comparison set of species and the final predictive set of species, as well as the models created to assess performance, were run using 1000 warmup iterations and 2000 sampling iterations, each on 4 chains for a total of 8000 draws per model run.
To mitigate issues surrounding initial values being rejected in the distance model (see Discussion), we set the initial values of EDR to be the mean EDR from NA-POPS estimates; for species where EDR estimates did not exist in NA-POPS, we used the overall mean EDR as initial value.
Cross-validation models were run with 1000 warmup iterations and 500 sampling iterations, each on 4 chains for a total of 2000 draws per model run.
All of the models made use of Stan's reduce\_sum functionality, which allows for within-chain parallelization as well as the usual across-chain parallelization.
This was particularly useful in our case as the multi-species removal model contained over 3 million data points and the multi-species distance model contained over 4 million data points, and so computational expense was a large concern.
All models were run on one of two multi-core processing servers, one running Ubuntu 20.04.4 LTS and one running Ubuntu 20.04.6 LTS.


\section{Results}
\subsection{Parameter Estimates}

\par We found much less support for a difference in cue rate between resident and migrant species than was found in \citet{solymos_evaluating_2018}. 
Mean overall log cue rate in the multi-species model was -1.09 (95\% credible interval of [-1.24, -0.94]); both estimated effects for resident and migrant species were effectively 0 (Figure \ref{fig:params}A).
We also found little support for migration strategy having any effect on EDR.
There is some evidence that a resident species may have a slightly higher EDR than a migrant species, but the 95\% credible intervals of both effects overlapped with the 95\% credible interval of the overall estimated intercept (Figure \ref{fig:params}B).
We did find similar support for change in EDR by body size and pitch, in that body size tended to have an increasing effect on EDR, and song pitch had a decreasing effect on EDR (Figure \ref{fig:params}B).

\begin{figure}[h!]
	\includegraphics{../output/plots/parameters_plot.png}
	\caption{Posterior distributions with 95\% credible intervals (shaded area) of parameters estimated in the multi-species removal model, with overall mean log cue rate as the intercept (A), migratory strategy as departures from the intercept based on whether a species was a resident (mu\_mig\_strat[1]) or migrant (mu\_mig\_strat[B]) (B), and extra variance term after accounting for the phylogenetic variation (C). Posterior distributions with 95\% credible intervals (shaded area) of parameters estimated in the multi-species distance model, with (A) overall mean log EDR as the intercept, with departures from the intercept modelled as migratory strategy (resident: mu\_mig\_strat[1], migrant: mu\_mig\_strat[B]), (B) slope parameters of changes in EDR given change in mass and pitch, and (C) extra variance term after accounting for traits.}
	\label{fig:params}
\end{figure}

\subsection{Comparison of Estimates between Single Species and Multi Species}

\par We found very little evidence for an overall change in cue rate for a given species modelled by a single-species model vs a multi-species model (Figure \ref{fig:1vs1}A).
Seven out of the 316 species considered for removal modelling had a percent change in cue rate of more than 20\%.
These species were Willow Ptarmigan (\textit{Lagopus lagopus}), Rusty Blackbird (\textit{Euphagus carolinus}), Red-bellied Sapsucker (\textit{Sphyrapicus ruber}), Common Redpoll (\textit{Acanthis flammea}), Calliope Hummingbird (\textit{Selasphorus calliope}), Bohemian Waxwing (\textit{Bombycilla garrulus}), and Allen's Hummingbird (\textit{Selasphorus sasin}).
All seven of these species were among the lowest cue rates in both the single-species and multi-species context (Figure \ref{fig:1vs1}A).

\par We found little evidence for a change in modelled EDR between single species and multi-species distance models (Figure \ref{fig:1vs1}B). 
(NEEDS UPDATING) Seven out of the 317 species had a percent change in EDR of more than 20\% (Figure \ref{fig:1vs1}B).
These species were Sharp-tailed Grouse (\textit{Tympanuchus phasianellus}), Sedge Wren (\textit{Cistothorus stellaris}), Ring-necked Pheasant (\textit{Phasianus colchicus}), Chimney Swift (\textit{Chaetura pelagica}), Boat-tailed Grackle (\textit{Quiscalus major}), and Baird's Sparrow (\textit{Ammodramus bairdii}).

\begin{figure}[h!]
	\includegraphics{../output/plots/1vs1.png}
	\caption{Modelled difference with 95\% credible interval (shaded area) between cue rates estimated from the multi-species model developed here, minus the single-species model in NA-POPS \citep{edwards_point_2023} (A); and 1-to-1 plot comparing cue rates estimated from the single-species model vs the multi-species model for each species (B). Modelled difference with 95\% credible interval (shaded area) between EDR estimated from the multi-species model developed here, minus the single-species model in NA-POPS \citep{edwards_point_2023} (A); and 1-to-1 plot comparing EDR estimated from the single-species model vs the multi-species model for each species (B).}
	\label{fig:1vs1}
\end{figure}

\subsection{Comparison of Precision between Single Species and Multi Species}

\par (NEEDS UPDATING) We found some evidence that a multi-species removal model improved the precision of cue rate estimates compared to estimates from a single-species model.
The intercept of standard deviations of cue rates estimated in a single species model was greater than the intercept of standard deviations of cue rates estimated in a multi-species model, with very little overlap in their 95\% credible intervals (Figure \ref{fig:sd}A).
This implies that the baseline standard deviation of cue rates is higher in a single species model than a multi species model for very low sample sizes.
However, the slope of standard deviations of cue rates estimated in a single species model was farther from 0 (i.e., more negative) than the slope of standard deviations of cue rates estimated in a multi-species model, with very little overlap in their 95\% credible intervals (Figure \ref{fig:sd}A).
This implies that the decrease in standard deviation as sample size increases is lesser in a multi-species model than a single-species model.
This indicates that after a certain sample size, it is likely that the difference in standard deviation between a single species and multi species model would be negligible.

\par (NEEDS UPDATING) We found little evidence that a multi-species model improves the precision of EDR compared to that of a single-species model.
The 95\% credible intervals for both estimated slopes and intercepts overlapped with each other, indicating that the standard deviations are effectively the same for a single-species and multi-species estimate as sample size increases (Figure \ref{fig:sd}A).

\begin{figure}[h!]
	\includegraphics{../output/plots/sd_comp.png}
	\caption{Posterior distributions with 95\% credible intervals (shaded areas) of (A) modelled intercepts of standard deviation for the single-species removal model (alpha[1]) and multi-species removal model (alpha[2]), and (B) modelled slopes of standard deviation with increasing sample size for the single-species removal model (beta[1]) and multi-species removal model (beta[2]). Posterior distributions with 95\% credible intervals (shaded areas) of (A) modelled intercepts of standard deviation for the single-species distance model (alpha[1]) and multi-species distance model (alpha[2]), and (B) modelled slopes of standard deviation with increasing sample size for the single-species distance model (beta[1]) and multi-species distance model (beta[2]).}
	\label{fig:sd}
\end{figure}

\subsection{Comparison of Predictive Accuracy between Single Species and Multi Species}

\par The cross validation revealed that single species model actually had better predictive capacity for more species (222 out of 316) than the multi-species model (94 out of 316), but the difference in predictive capacity was overall very small.
Mean overall difference in posterior predictive density was $-2.21 \times 10^{-5}$.
Single-species models tended to have better predictive capacity when sample size was low (again noting a very small difference), and a negligible difference as sample size increased (Figure \ref{fig:cv}A).
There did not appear to be any patterns in model selection among phylogeny or traits (see Supplemental Figure 1 for full phylogenetic tree).
Six out of the 7 species that had a percent change in cue rate of more than 20\% had the multi-species model chosen as the better predictive model.
Common Redpoll was the only species that had the single-species model chosen (but please see Discussion about this particular species).

\par The cross validation revealed that the multi-species model tended to have slightly more support than the single-species model in terms of predicting EDR.
However, similar to the removal model cross-validation results, the overall difference in posterior predictive density was very small, with a mean difference of $1.85 \times 10^{-4}$.
Multi-species models tended to have better predictive capacity when sample size was low (again noting a very small difference), and a negligible difference as sample size increased (Figure \ref{fig:cv}B).
Interestingly, only two (Chimney Swift and Boat-tailed Grackle) of the six species with a greater than 20\% change in EDR had the multi-species model chosen as the better predictive model.

\begin{figure}[h!]
	\includegraphics{../output/plots/cv_plot.png}
	\caption{Mean difference in pointwise posterior predictive density. Points are mean difference by species. Difference was calculated as multi-species minus single-species, and so positive differences (coloured in red for clarity) represent a multi-species model preference, whereas negative differences (coloured in blue for clarity) represent a single-species model preference. Regression line shows the relationship between mean difference and species sample size. Mean difference in pointwise posterior predictive density. Points are mean difference by species. Difference was calculated as multi-species minus single-species, and so positive differences (coloured in red for clarity) represent a multi-species distance model preference, whereas negative differences (coloured in blue for clarity) represent a single-species distance model preference. Regression line shows the relationship between mean difference and species sample size.}
	\label{fig:cv}
\end{figure}

\subsection{Predictions of Cue Rate and EDR for Undersampled Species}

\par We used the multi-species removal model and distance model to generate predictions of cue rate and EDR, respectively, for 7 species of undersampled birds from the original NA-POPS analysis (6 for distance modelling because Tricoloured Blackbird had sufficient distance sampling data).

\par The multi-species removal model produced reasonable estimates of cue rates for all seven species of bird considered here (Figure \ref{fig:predictions}A).
Species where the sample size requirements were relaxed (i.e., all except Bicknell's Thrush and Kirtland's Warbler) had reasonable precision associated with the estimates.
One exception to this was LeConte's Thrasher, where the precision was rather low.
The species for which we had no data (i.e., Bicknell's Thrush and Kirtland's Warbler) had estimates of cue rate similar to that of their closely-related species, demonstrating the shrinkage effect of the multi-species model.

\par The multi-species distance model also produced reasonable estimates of EDR for all six species of landbird considered here (Figure \ref{fig:predictions}B).
Species where the sample size requirements were relaxed (i.e., all except Bicknell's Thrush) had fairly precise estimates, except for LeConte's Thrasher and Lesser Prairie Chicken where the estimated EDR spanned greater than 50 m in either direction.
The species for which we had no data (i.e., Bicknell's Thrush) had an estimate of EDR similar to what would be expected by a bird of its size, song pitch, and habitat association, demonstrating the shrinkage effect of the multi-species model.

\begin{figure}[h!]
	\includegraphics{../output/plots/predictions_figure.png}
	\caption{Cue rates (points), 50\% credible interval (bolded line), and 95\% credible interval (thin line) for seven species of rare and undersampled North American landbirds. Species are 4 letter banding codes: SPOW = Spotted Owl (n = 7), BITH = Bicknell's Thrush (n = 0), LCTH = LeConte's Thrasher (n = 1), TRBL = Tricoloured Blackbird (n = 74), HASP = Harris's Sparrow (n = 65), KIWA = Kirtland's Warbler (n = 0), and LEPC = Lesser Prairie-chicken (n = 45). Effective detection radius (points), 50\% credible interval (bolded line), and 95\% credible interval (thin line) for six species of rare and undersampled North American landbirds. Species are 4 letter banding codes: SPOW = Spotted Owl (n = 28), BITH = Bicknell's Thrush (n = 0), LCTH = LeConte's Thrasher (n = 1), HASP = Harris's Sparrow (n = 13), KIWA = Kirtland's Warbler (n = 1), and LEPC = Lesser Prairie-chicken (n = 45).}
	\label{fig:predictions}
\end{figure}

\section{Discussion}

\par In this paper, we developed a multi-species model of detectability by expanding upon the QPAD methodology developed by \citet{solymos_calibrating_2013}.
We applied the multi-species QPAD removal model and multi-species QPAD distance model to the large dataset of point counts collected by the NA-POPS project \citep{edwards_point_2023}, and compared estimates of cue rate and EDR from a multi-species context to estimates of cue rate and EDR from the previous single species context.
Given the results of these comparisons, we predicted these components of detectability for seven species of North American landbird that are rare and undersampled.

\subsection{Model Performance}

\par The multi-species QPAD Distance model had several issues related to initialization, sampling, and convergence throughout this study.
For many iterations of the model development, Stan's generation of initial values proved difficult.
For all unconstrained parameters, Stan draws initial values from a uniform(-2,2) distribution.
On the scale that we modelled detection distances (i.e., log detection distance in hundreds of metres), these are reasonable values.
At the low end, an initial value of -2 would correspond to a detection distance of $e^{-2} \times 100 = 13.5$ m, which is low but somewhat plausible for a small-bodied, high-pitched species such as a hummingbird.
At the high end, an initial value of 2 would correspond to a detection distance of $e^{2} \times 100 = 739$ m, which is high but still somewhat plausible for large-bodied, low-pitched species like a grouse or prairie-chickens.
However, where Stan seemed to struggle is when these randomly assigned values for detection distances were effectively impossible for a large set of species.
That is to say, if the values were initialized such that hummingbirds had a detection distance of 500 m or more, and grouse or prairie-chickens had a detection distance of 15 m or less, this would result in an impossible likelihood, forcing Stan to reject the initial values.
Our fix was to set initial values to be jittered mean detection distances originally estimated in \citet{edwards_point_2023}; for species without estimates, we used the mean EDR for all species.
This approach seemed to work as the initial values were no longer rejected, but we caution that this could very well be a "bandaid solution" to an underlying issue with the model itself.

\par The distance models also seemed to struggle with sampling the posterior distribution, in that over a quarter of the iterations resulted in a divergent transition \citep{betancourt_diagnosing_2016, leimkuhler_simulating_2005}.
This is likely due to hierarchical nature of the model and the parameterization choices that we made.
\citet{betancourt_hierarchical_2020} discuss at length some issues that arise when sampling from hierarchical models in cases where the hierarchical groups are data rich, where a "centred parameterization" should be used, or data poor, where a "non-centred parameterization" should be used.
We experimented with using only a non-centred parameterization which resulted in extremely low effective sample sizes (as low as 4 for 8000 iterations), large R-hat values, and took an unreasonable amount of time to finish sampling (approximately 2.5 weeks).
The model presented in this paper is a centred parameterization, which produced much more reasonable effective sample sizes and R hat values, finished within a day and a half, but still suffered from degeneracies such as hierarchical funnelling as described by \citet{betancourt_hierarchical_2020}.
In our case, because we are modelling on a hierarchy that contain many data points for some species x trait groups and fewer data points for other species x trait groups, our divergent transitions may be resolved if we were to implement our model using a mixture of centred and non-centred parametrization as described by \citet{betancourt_hierarchical_2020}.

\par For removal modelling, we experience no issues related to initialization, sampling, or convergence. 
The model finished within approximately 24 hours, and all parameters had R-hat values of 1, or within 0.001 of it.
We note that even though we used a Gaussian process to estimate cue rate, we did not have to estimate the covariance matrix used for the Gaussian process, because it was effectively a known parameter provided by the phylogenetic relationship.
This would account for the relatively quick performance of this model given the number of parameters to estimate and the amount of data used; had the covariance matrix needed to be estimated, this model would have likely taken much longer to complete.

\subsection{Removal Modelling}
\par Estimates of cue rate changed very little from species that were originally modelled by the single-species models in \citet{edwards_point_2023}, to the same ones modelled in the multi-species model here.
The 5-fold cross validation tended to show no preference for one model or another when sample size was large, and a preference (albeit an extremely small preference) for single-species models when sample size was smaller.
When sample size was small, a multi-species model tended to produce more precise estimates of cue rate.
Given the extremely small preference for single-species models under a lower sample size, it would appear that the bias-variance trade-off is likely a welcome trade-off here, because the decrease in variance is met by only a very slight increase in bias.

\par There were seven species where the relative difference in cue rate between the single-species model and multi-species model was over 20\% (noting that this threshold was chosen arbitrarily).
These species (Willow Ptarmagin, Rusty Blackbird, Red-breasted Sapsucker, Common Redpoll, Calliope Hummingbird, Allen's Hummingbird, and Bohemian Waxwing), mostly had fairly small sample sizes ($\leq 515$ for all species except Red-breasted Sapsucker [$n = 1854$] and Common Redpoll [$n = 2009$]), had the lowest cue rate in the single-species models, and had among the lowest cue rates in the multi-species model.
For removal modelling, all those species except for Common Redpoll ended up having the multi-species model chosen as the more predictive model, indicating that sharing information among related species tended to improve their estimates of cue rate.
Common Redpoll was likely an exception to this because the original estimate of cue rate in NA-POPS was a wild underestimate (cue rate of $5.9\times 10^{-9}$), likely because of a convergence issue in the \texttt{detect} package.
It could be worth running Common Redpoll using the Bayesian single species model developed here to see if the estimate of cue rate is still extremely low, or if the vague but still somewhat informative prior helps to bring the estimated parameter into a more sensible space.
It could also be worth investigating the merit of this type of modelling to estimate cue rate for species where the cue rate is extremely low.
In our case, 41 out of the 79 species that had the lowest 25\% quantile of cue rate in the single-species model ended up having the multi-species model chosen as the preferred model.

\par Although this was not a primary goal of our study, we found much less support than \citet{solymos_phylogeny_2018} for migration strategy having an effect on cue rate.
Both the estimate effects for resident species and migratory species were effectively 0, with very little of the density above or below 0.025 from 0.
One potential reason is that we had proportionally more resident species in our analysis (85 out of 316; 26.9\%) compared to \citet{solymos_phylogeny_2018} (23 out of 141; 16.3\%), and so the richer information provided by these additional species could have brought the effect down to 0.
Additionally, while \citet{solymos_phylogeny_2018} only regressed estimates of cue rate against the predictor variables, we actually estimated cue rate within the context of the same predictors, and so the phylogenetic effect could have just overwhelmed the estimates, making the migratory strategy meaningless.

\subsection{Distance Modelling}
\par Estimates of EDR also changed very little, albeit a non-zero amount, from species that were originally modelled by the single-species models in \citet{edwards_point_2023}, to the same ones modelled in the multi-species model here.
However, there was evidence for an average small downward shift in EDRs when modelled under a multi-species context.
Still, the 5-fold cross validation tended to show almost no preference for one model or another when the sample size was large, and a slight (but still very small) preference for the multi-species model when the sample size was small.
There was less evidence for the multi-species model being able to improve the precision of EDR estimates across sample sizes, indicating that a single-species model may mostly be sufficient for precise estimates.
Under this context, it would appear that a multi-species model may slightly bring down estimates of EDR for species with a smaller sample size.

\par There were six species where the relative difference in EDR between the single-species model and multi-species model was over 20\%. 
Unlike removal modelling, these species (Sharp-tailed Grouse, Sedge Wren, Ring-necked Pheasant, Chimney Swift, Boat-tailed Grackle, and Baird's Sparrow) had little in common with each other in terms of sample size (range of 227 for Boat-tailed Grackle through 4466 for Baird's Sparrow) and EDR (ranging from 50.8 m to 226 m in single-species, and from 73.2 m to 159.1 m in the multi-species model). 
Additionally, there was mixed support among these species for single-species vs. multi-species: Chimney Swift and Boat-tailed Grackle had the multi-species model preferred, whereas the others had the single-species model preferred.
It could be noted that both Chimney Swift and Boat-tailed Grackle showed an increase in estimated EDR from single-species to multi-species modelling and preferred the multi-species model, but Baird's Sparrow also showed an increase and preferred the single-species model.

\par Similar to removal modelling, we found less support than \citet{solymos_phylogeny_2018} for migration strategy having an effect on EDR, although the posterior distribution for both the resident and migrant species were much further away from zero, indicating at least some support.
Habitat type, body mass, and song pitch, as expected, all had an overall effect on EDR, as was found in \citet{solymos_phylogeny_2018} and \cite{johnston_species_2014}.

\subsection{Relaxation of Sample Size}

\par For five out of the seven species where we generated new predictions of detectability in this study, we were interested in seeing how a multi-species QPAD removal model could estimate cue rate with few data, and how a multi-species QPAD distance model could estimate EDR with few data.

\par Overall, the estimates of cue rate fell roughly where we would expect based on related species, for species where we were simply relaxing the sample size requirements.
For example, Harris's Sparrow (n = 65) had an estimated cue rate of 0.30 cues per minute, with a 95\% credible interval of [0.21, 0.42], compared to the overall mean cue rate of 0.37 cues per minute for all 32 other Passerellidae species in NA-POPS.
Similarly, LeConte's Thrasher (n = 1) had an estimated cue rate of 0.32 cues per minute, with a 95\% credible interval of [0.18, 0.57].
Though this was a much larger interval than Harris's Sparrow, it still falls within a reasonable distance from the overall mean cue rate of 0.36 cues per minute for all 6 other Mimidae species in NA-POPS.

\par Estimates of EDR also fell roughly where we would expect based on species of similar traits, for the species where we were relaxing samples sizes.
For example, Harris's Sparrow (n = 13) had an estimated mean EDR of 65.6 m, with a 95\% credible interval of [53.1, 82.5].
This range does fall below the mean EDR of 85.3 m for the other sparrows in NA-POPS, but Harris's Sparrow song pitch of approximately 3.8 Hz falls above the average song pitch of 3.0 Hz for the other sparrows in NA-POPS, which could contribute to the smaller EDR.
On the other hand, LeConte's Thrasher (n = 1) had an estimated EDR of 134.2 m, with a 95\% credible interval of [81.8, 230.5], which falls roughly in line with the mean EDR of 113.2 m of the six other thrashers in NA-POPS.

\par Of course, it is likely that the width of the credible intervals (i.e., the precision) of these relaxed sample-size species are going to depend both on the number of data points available for that particular species, the number of "similar" species, and the number of data points available for those similar species.
However, species that do not have many other similar species from which to borrow information still have the benefit of using information from the priors that we set in the model, which is a large advantage to modelling detectability in a Bayesian context.
Additionally, it is important to note that we did not account for other sources of variation at the survey level, such as when the survey was conducted (i.e., time of day, time of year), or where the survey was conducted (e.g., in a forest, by a roadside).
It is well known that the ability to detect a species can be very dependent on the time of day or year that an observer is surveying, as well as the environmental conditions of the survey \citep{edwards_point_2023}.
Thus, some of the variation in these estimates of cue rate and EDR could be due to not accounting for survey-level variation that affects detectability, and so it is important that future iterations of this model incorporate both species-level effects on detectability \textit{and} survey-level effects on detectability.

\subsection{Predicting Detectability for Species with No Data}

\par We generated predictions of cue rate and EDR for both Bicknell's Thrush and Kirtland's Warbler.
For both species, we had zero removal sampling data, and so cue rates were solely based off information sharing among phylogeny (and to a much lesser extent, migration status).
For Kirtland's Warbler, we did have one data point for distance sampling, so predictions of EDR for Kirtland's Warbler here is not entirely based off zero data points.
Bicknell's Thrush, however, had zero data points, and so EDRs for this species were based solely off information sharing among other species with similar traits.

\par As expected, the predicted cue rate for Bicknell's Thrush (0.28 cues per minute) and the predicted cue rate for Kirtland's Warbler (0.29 cues per minute) fell very close to the overall mean cue rate of 0.27 for all 11 other Turdidae species and 0.31 for all 44 other Parulidae species in NA-POPS, respectively.
That is, in the absence of any data, the estimated cue rate of Bicknell's Thrush and Kirtland's Warbler was shrunk toward cue rates of most similar species.

\par For distance modelling, the predicted EDR for Bicknell's Thrush (73.1 m) and the predicted EDR for Kirtland's Warbler (75.0 m) fell reasonably close to the overall mean EDR of 92.7 m for all other 11 Turdidae species and 61.2 m for all 44 other Parulidae species in NA-POPS, respectively.
That is, in the absence of any data (or with very little data in the case of Kirtland's Warbler), the estimated EDR of Bicknell's Thrush and Kirtland's Warbler was similar to those of other Thrushes and Warblers, respectively, but with departures from overall mean based on specific traits.

\par In both the removal modelling and distance modelling, the magnitude of contribution of data from related species appeared to play a role in the width of the credible intervals.
For removal modelling, although both credible intervals were fairly wide, Bicknell's Thrush had a 95\% credible interval of [0.16, 0.48], representing an approximate 53\% change on either end, whereas Kirtland's Warbler had a 95\% credible interval of [0.18, 0.48], representing an approximate 49\% change on either end.
For distance modelling, both credible intervals were also fairly wide, but Bicknell's Thrush had a 95\% credible interval of [40.4, 135.0], representing an approximate 58\% change on either end, whereas Kirtland's Warbler had a 95\% credible interval of [46.6, 124.8], representing an approximate 47\% change on either end.
Although the difference in width of the intervals is small, this could be due to the greater number of other species contributing information to Kirtland's Warbler estimates than to Bicknell's Thrush.
In the case of distance modelling, the one data point contributing to Kirtland's Warbler's EDR could have also played a part.
With that in mind, it would likely be important to consider how many "related" species are available with information (and how many data points do each of those related species have) when deciding on making a prediction of cue rate.

\subsection{Comparisons of Predicted Detectability to Other Studies}

\par Using the predictions of cue rate and EDR for Bicknell's Thrush and Kirtland's Warbler, we can calculate availability and perceptibility, and hence detectability, for a given survey type, and compare to previous estimates of detectability obtained from other studies.
As a reminder, following derivations in \citet{solymos_calibrating_2013}, detectability is the product of availability and perceptibility.
Availability $p(t)$ is defined as the probability of a bird giving a cue during a survey run for $t$ minutes, and is calculated as
$p(t) = 1 - e^{-t\phi}$
where $\phi$ is the cue rate of the species.
Perceptibility $q(r)$ is defined as the probability of an observer hearing or seeing a bird, given the bird gives a cue, with a survey radius of $r$ metres, and is calculated as
$q(r) = \dfrac{\tau^2 \left\{1 - e^{-\dfrac{r^2}{\tau^2}}\right\}}{r^2}$
where $\tau$ is the EDR of the species.

\par For Bicknell's Thrush, using the equations above, the predicted cue rate of 0.28, predicted EDR of 73.1 m, and an example survey design of 5 minutes recording all birds within 100 m, we estimate a detectability of 0.34.
We can also use the lowest predicted cue rate and EDR and the highest predicted cue rate and EDR (based on the respectively credible intervals) to get a range of detectabilities from 0.09 through 0.70.
\citet{aubry_bicknells_2018}, however, estimate detection probabilities of $\geq$ 0.74 and $\leq$ 0.88, and so even our highest potential value of detectability falls short of their estimated range.
However, we note that \citet{aubry_bicknells_2018} are using data that specifically target Bicknell's Thrush in their range, and they are using surveys conducted at peak Bicknell's Thrush breeding time, and so it is likely that detectability will be high in that specific case.

\par For Kirtland's Warbler, using the equations above, the predicted cue rate of 0.29, predicted EDR of 75.0 m, and an example survey design of 5 minutes recording all birds within 100 m, we estimate a detectability of 0.36.
Similar to our Bicknell's Thrush example, we can also use the lowest predicted cue rate and EDR and the highest predicted cue rate and EDR (based on the respective credible intervals) to obtain a range of detectabilities from 0.13 through 0.67.
A study by \citet{van_dyke_comparative_2022} found estimated detection probabilities for Kirtland's Warbler that ranged from effectively 0, through to a detection probability of 0.73.
In that study, they compared estimates of detectability among different jack pine stands and red pine stands, where the detectability was highest in jack pine stands than the red pine stands.

\subsection{Improvements to this model}

Both examples in the previous section highlight the need for future iterations of this multi-species model to consider environmental and temporal covariates that affect peaks in detectability through the year and for different environmental conditions.
This is especially true for the types of species we are interested in generating predictions for, i.e., rare and/or undersampled species.
In general, rare species will prefer rare habitats, or will at least be confined to a very small range (as is the case with Bicknell's Thrush and Kirtland's Warbler above).
Thus, while we can generate reasonable estimates of the components of detectability (i.e., availability through cue rate and perceptibility through EDR) based on information borrowed from similar species (either through phylogeny or traits), this model as it stands may fail to glean more specific information that takes into account detectability peaks throughout the year, or detectability peaks given specific habitats.
With this in mind, it may be beneficial to include temporal covariates at the survey level, similar to what was done in \citet{solymos_calibrating_2013, solymos_evaluating_2018, edwards_point_2023}, so that further information about cue rate may be shared among similar species giving cues at similar times of day or year.
Similarly, it may be beneficial to include environmental covariates at the survey level, similar to what was done in \citet{solymos_calibrating_2013, edwards_point_2023}, so that further information about EDR may be shared among similar species being surveyed at similar habitats.
This could also provide an opportunity to disentangle species x survey interactions in detectability.

\par The examples above additionally highlight an excellent opportunity to make use of a powerful feature of Bayesian models, that being the inclusion of expert knowledge as prior information.
For example, if a Bicknell's Thrush expert knows that there is something particular about how often they sing or give a cue compared to other thrushes, or how disproportionately quiet or loud they are compared to species with similar traits, this can be coded directly into the prior distribution.
Similarly, this may also provide a unique opportunity to make use of traditional ecological knowledge to further inform predictions of detectability.
For example, in the Anishinaabe language, bird taxonomy is based both on physical features, but also behaviours such as flight, calls, morning rituals, and weather \citep{pitawanakwat_evening_2022}.
This taxonomy is based off observations of bird behaviour that has been passed down through generations, and may vary slightly (or heavily) from taxonomy based on genomics.
In fact, it could be argued that a taxonomy based on how humans observe birds may actually be more relevant for detectability, and so a future opportunity could be to adopt a two-eyed seeing approach \cite{reid_twoeyed_2021} to predicting detectability in rare birds.

\subsection{Conservation Implications}
\par Detectability often goes unaccounted for in conservation problems, often because the estimates of detection probability simply do not exist, or they are extremely uncertain \citep{bennett_how_2023}.
This model provides an excellent opportunity for detection probabilities to be estimated with higher precision for several species of landbirds in North America, and also provides an opportunity to predict detection probabilities for species which have very little (or even no) data.
This means that conservation problems that rely on estimates of detectability such as Value of Information analyses \citep{canessa_when_2015, bennett_when_2018} or prioritizations \citep{hanson_prioritizr_2023} can look to make more accurate decisions.
For organizations such as Partners in Flight, Boreal Avian Modelling project (\citep{cumming_toward_2010}), and others that rely on detectability information for bird population size estimates, these improved estimates (and new estimates and predictions) provide a way forward for continuing to produce accurate estimates of population size that are based on data-driven detectability estimates.

\par The model presented here can be used to fill out the remaining missing species in the NA-POPS database, in order to estimate detectability for every species of North American landbird.
By making use of phylogeny and trait information, bird species with fewer than 75 data points can have estimates generated based on the few data points available, and information available from other species.
As shown here, birds with no data points can have detectability predicted that falls closely in line with similar species.
We caution that birds that are to have detectability predicted should be noted as such on any communications, such that organizations know that the estimate is simply a "best guess" based on other similar species.

\par Finally, we see this model as a proof-of-concept for use with other taxa.
Obviously, given the vast amount of bird data available, it was easy enough to obtain estimates for enough data-rich species such that the few data-poor species could have enough information to borrow.
However, one benefit that we have shown here is that Bayesian models can still come up with reasonable predictions if there are informed enough priors on which to fall back.
That is to say, for a taxon such as butterflies where there are some species with data and others with very little data, but there is overall very little data in general \citep{lewthwaite_geographical_2022} with reasonably-informed priors, a researcher may still be able to glean reasonable estimates of detectability with the little data available.
We suggest that this type of model where information about detectability is shared among species be tested with taxon that generally have fewer data collected.

\bibliography{refs}

\end{document}
